### PR TITLE
Exclude GetStackTraceALotWhenPinned test on zLinux & xMac

### PR DESCRIPTION
**Pull request by fengxue-IS**
https://github.com/eclipse-openj9/openj9/issues/18908

### CODE REVIEW

**Comment by fengxue-IS** [comment]
@pshipton can you please take a look

**Comment by pshipton** [comment]
It failed on xmac as well, now that we have xmac machines working again. Pls exclude it there as well.

**Comment by fengxue-IS** [comment]
Updated

File: openjdk/excludes/ProblemList_openjdk21-openj9.txt [hunk 1]
```diff
@@ -116,6 +116,7 @@ java/lang/System/LoggerFinder/modules/UnnamedLoggerForImageTest.java https://git
 java/lang/Thread/SleepSanity.java https://github.com/eclipse-openj9/openj9/issues/17638 windows-all
 java/lang/Thread/UncaughtExceptions.sh https://github.com/eclipse-openj9/openj9/issues/6690 generic-all
 java/lang/Thread/virtual/StackTraces.java https://github.com/eclipse-openj9/openj9/issues/16045 generic-all
+java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java https://github.com/eclipse-openj9/openj9/issues/18908 linux-s390x,macosx-x64
```
Comment (pshipton) [inline openjdk/excludes/ProblemList_openjdk21-openj9.txt:119]: I believe the `#id0` part is required.

Comment (fengxue-IS) [reply openjdk/excludes/ProblemList_openjdk21-openj9.txt]: Fixed

**Review by pshipton — APPROVED** [review]

**Pull request merged**